test/e2e/snaps/test-snap-installed.spec.js

```diff
@@ -71,25 +76,30 @@ describe('Test Snap Installed', function () {
         // click to connect to errors snap
         const errorButton = await driver.findElement('#connecterrors');
         await driver.scrollToElement(errorButton);
-        await driver.delay(500);
+
+        // added delay for firefox (deflake)
+        await driver.delay(1000);
+
+        // wait for and click connect
+        await driver.waitForSelector('#connecterrors');
```

Comment:
I see you're doing this in some other tests so I'll just comment once. Why do we need to have both the `waitForSelector` call here with an increase in the delay?

Reply:
Well, because at first, it didn't hurt. In the case now, because those delays are "firefox specific" we need them for chrome instances.